mods/noita-mp/files/scripts/noita-components/name-tags.lua

```diff
@@ -0,0 +1,23 @@
+
+function PlayerNameFunction(playerName)
+    gui = gui or GuiCreate()
+    GuiStartFrame(gui)
+    local x1, y1 = GuiGetScreenDimensions(gui)
+    x1, y1 = x1/2, y1/2
+
+    local entityId = GetUpdatedEntityID()
+    local x, y = EntityGetTransform(entityId)
+
+    function getEntityPositionOnScreen(entityId)
```

Comment:
Here you can see, that I already learned something.

Didnt know that it would be possible to define a function in a function.
Looks like private and public access modifiers-